GeneratorInterface/Hydjet2Interface/interface/Hydjet2Hadronizer.h

```diff
@@ -1,49 +1,35 @@
-/*                                                                            
- Based on class InitalStateHydjet:                                          
- Nikolai Amelin, Ludmila Malinina, Timur Pocheptsov (C) JINR/Dubna
- amelin@sunhe.jinr.ru, malinina@sunhe.jinr.ru, pocheptsov@sunhe.jinr.ru 
- November. 2, 2005                                
-
-*/
-
 #ifndef Hydjet2Hadronizer_h
 #define Hydjet2Hadronizer_h
 
-/** \class Hydjet2Hadronizer
- *
- * Generates HYDJET++ ==> HepMC events
- *
- * Andrey Belyaev
- * for the Generator Interface. Sep 2014
- *********************************************/
-
-#include "DatabasePDG.h"
-#include "Particle.h"
-#include "InitialState.h"
-
-#define PYCOMP pycomp_
+/**
+ *    \class HydjetHadronizer
+ *    \brief Interface to the HYDJET++ (Hydjet2) generator (since core v. 2.4.2), produces HepMC events
+ *    \version 1.0
+ *    \author Andrey Belyaev
+ */
 
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
 #include "GeneratorInterface/Core/interface/BaseHadronizer.h"
 
-#include "CLHEP/Random/RandomEngine.h"
-#include "CLHEP/Random/RandFlat.h"
-#include "CLHEP/Random/RandPoisson.h"
-#include "CLHEP/Random/RandGauss.h"
+#include "FWCore/Framework/interface/ConsumesCollector.h"
+#include "FWCore/Framework/interface/EDProducer.h"
+#include "FWCore/Utilities/interface/EDGetToken.h"
+#include "SimDataFormats/CrossingFrame/interface/MixCollection.h"
+#include "SimDataFormats/PileupSummaryInfo/interface/PileupMixingContent.h"
+
+//#include "CLHEP/Random/RandomEngine.h"
+//#include "CLHEP/Random/RandFlat.h"
+//#include "CLHEP/Random/RandPoisson.h"
+//#include "CLHEP/Random/RandGauss.h"
```

Comment:
a long the lines of Phats comment: wouldn't it make sense to remove the CLHEP part as well?

Reply:
Thanks, fixed.